code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -898,23 +901,64 @@
 	overlay_state = "tags"
 	drop_sound = 'sound/items/drop/accessory.ogg'
 	pickup_sound = 'sound/items/pickup/accessory.ogg'
-	var/can_be_broken = FALSE
+	gender = PLURAL
+	var/can_be_broken = TRUE
 	var/separated = FALSE
 	var/tag_type = /obj/item/dogtag
 
+	var/dogtags_name = DOGTAGS_UNSET
```

Comment:
this should just be `var/dogtags_name` so that it's null

you can then just check for `if(dogtags_name && ...)` below